single_tech_samples/databricks/sample4_ci_cd/pyspark/README.md

```diff
@@ -1 +1,40 @@
-# Readme
+# Databricks CI/CD - MLflow Experiment
+
+This sample demonstrates how to deploy MLflow Experiments/Jobs to Azure Databricks as well as how to do unit test and integration test via CI/CD pipeline.
+
+Note that we use dbx CLI to deploy MLflow experiments/jobs,  for more information, please see [this repo](https://github.com/databrickslabs/cicd-templates).
+
+## How to use this sample
+
+### Prerequisites
+
+- [Azure DevOps account](https://azure.microsoft.com/en-au/services/devops/)
+- [Github account](https://github.com/)
+- [Azure Databricks workspace](https://azure.microsoft.com/en-au/services/databricks/) with an [existing cluster](https://docs.microsoft.com/en-us/azure/databricks/clusters/create) running.
+
+### Setup and deployment
+
+1. Clone (or import) this repo.
+1. In Azure DevOps, [define a Variable Group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml) called `mdwdo-dbx-pyspark-test` with the following variables:
+    - **databricksHost** - The databricks host in this format: `https://adb-123456789xxxx.x.azuredatabricks.net`
+    - **databricksToken** - Databricks PAT token.
+    - **databricksClusterId** - Databricks cluster id that will be used to run the notebooks.
```

Comment:
Thanks Lace! Sorry for this copy-paste issue :-D

Since we're using Key Vaults instance to store those 3 values now, above bullet point has been removed, instead we've added provisioning works from @meghapatilcode  to **[Prerequisite](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/single-tech/databricks-ops-thurstonchen/single_tech_samples/databricks/sample4_ci_cd/pyspark#prerequisites)** section and mentioned them there instead.